### DISCUSSION TITLE
Possible Algorithms not in Sk-Time?

### DISCUSSION THREAD

**Discussion by BradKML** [alan-turing-institute/sktime#3533] [General]
Just noticed that not every algorithm is included in sktime, and there are other libraries that has bigger models list.
https://github.com/winedarksea/AutoTS/blob/master/autots/models/model_list.py

e.g. Things like GLS, GLM, VECM, MAR, RRVAR and others, although fast, are not in the repo currently.
Would it be good to have a list of time series forecaster algos for reference in discussion?

**Comment by fkiraly** [comment]
sure! If you would like to open a wishlist with models to interface as an issue, possibly with reference to other packages that already have them, that would be much appreciated! Bonus points if you check which models are not yet in sktime.

In `sktime`, we would need to tag models and expose the parameters cleanly, and once we have a "wishlist", people can pick these up one-by-one.

**Reply by fkiraly** [reply]
PS: VECM is in sktime

**Reply by BradKML** [reply]
Thanks for the notice, will make this into a table-esque (or some other format?)
